initial_pose:
  x: 1.676129708308764
  y: 0.8701770648829898
  a: 1.2096520446054357
